src/site/_includes/components/Instruction.js

```diff
@@ -25,7 +27,7 @@ const capitalize = require('../../_filters/capitalize');
  * @param {string} listStyle The list style to use. Defaults to 'ul'.
  * @return {string} A list of instructions.
  */
-module.exports = (type, listStyle = 'ul') => {
+module.exports = (type, listStyle = 'ul', locale = defaultLocale) => {
```

Comment:
Let's avoid changing the way this function is called (eg by adding new parameters). You can use getLocaleFromPath to get locale instead, as shown here: https://github.com/GoogleChrome/web.dev/blob/main/src/site/_includes/components/Aside.js#L35

Please note that for this to work you need to change the notation of the component definition from arrow  function for "function() {...}"

Reply:
Oh!!! That's the trick. Thanks. Will update it now.